pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -1527,6 +1527,14 @@ in
     nvimRequireCheck = "null-ls";
   };
 
+  NotebookNavigator-nvim = super.NotebookNavigator-nvim.overrideAttrs {
+    dependencies = with self; [
+      mini-comment
+      hydra-nvim
```

Comment:
Did removing these throw errors at runtime? It looks like hydra is a protected call: https://github.com/GCBallesteros/NotebookNavigator.nvim/blob/20cb6f72939194e32eb3060578b445e5f2e7ae8b/lua/notebook-navigator/init.lua#L36

Reply:
I just checked and yes, the test still passes without any dependency. I'll remove them.